pkg/relay/ethereum/config.go

```diff
@@ -79,6 +79,11 @@ func (c ChainConfig) Validate() error {
 			}
 		}
 	}
+	for i, path := range c.AbiPaths {
+		if isEmpty(path) {
```

Comment:
Should we also check if the directory path exists?

Reply:
I don't think so, because:
- `ChainConfig::Validate` is meant to check the rule independent of runtime environements.
- Anyway `NewChain` will fail if event one of the directory paths doesn't exist.